corehq/apps/receiverwrapper/rate_limiter.py

```diff
@@ -42,7 +42,7 @@
                 per_second=1,
             ),
         ),
-    ).get_rate_limits
+    ).get_rate_limits(domain)
```

Comment:
nit: This is far away (15 lines) from the `lambda` where the parameter is declared. What do you think of restructuring this code (possibly in a normal `def` function) to make the scope more obvious?